packages/bee-q/src/components/panel/scss/bq-panel.scss

```diff
@@ -0,0 +1,20 @@
+/* -------------------------------------------------------------------------- */
+/*                                Panel styles                */
+/* -------------------------------------------------------------------------- */
+
+@import './bq-panel-variables';
+
+:host {
+  @apply block;
+}
+
+.bq-panel {
+  @include animation-fade-in;
+  @include hide-scrollbar;
```

Comment:
I would recommend not hiding the scrollbar, the users may not notice that there are other options down in the list... Please keep a line break between `@include` and `@apply`.